### ISSUE TITLE
HitBTC cancel order does not work

### ISSUE THREAD

**Issue by TearsOfTheEarth** [kroitor/ccxt#125]
Hi,

Not sure if this ever worked, trying to cancel order with:
hitbtc.cancel_order(111111)
And the ID is certainly valid as I see it on the web.
POST http://api.hitbtc.com/api/1/trading/cancel_order?nonce=1503164665&apikey=*&clientOrderId=111111
Response: {'Content-Type': 'application/x-www-form-urlencoded', 'X-Signature': '*', 'User-Agent': '*', 'Accept-Encoding': 'gzip, deflate'} {"CancelReject":{"clientOrderId":"111111","cancelRequestClientOrderId":"*","**rejectReasonCode":"orderNotFound"**}} 

And from the documentation:
https://hitbtc.com/api#cancelorder
It seems that some required parameters are missing, such as _symbol_. What do you think?

**Comment by kroitor** [comment]
That's strange, because in their docs they say, that the only required parameter is the clientOrderId...

<img width="1058" alt="screen shot 2017-08-19 at 20 58 07" src="https://user-images.githubusercontent.com/1294454/29489100-433bf9fc-8521-11e7-9c47-b85fab0983bd.png">

I'll add some debugging into this, just a second.

**Comment by kroitor** [comment]
Ok, I got it. Looks like it adds all params to the URL instead of putting the clientOrderId into the body. Will fix it very soon.

**Comment by TearsOfTheEarth** [comment]
There are two thing that confuses me:
1. I can see in the documentation which params are mandatory O_o
2. It returns clientOrderId in the error and that means that exchange gets it

**Comment by kroitor** [comment]
@TearsOfTheEarth are you sure that you're sending the clientOrderId and not the cancelRequestClientOrderId ?

**Comment by TearsOfTheEarth** [comment]
Absolutely, even created a standalone script with a single command and copy pasted "Order ID" from HitBTC's terminal.

**Comment by kroitor** [comment]
@TearsOfTheEarth I'm sorry what is the HitBTC terminal? I mean, can you create a HitBTC order with ccxt and then send its id to the cancel_order with ccxt ?

**Comment by TearsOfTheEarth** [comment]
This is how they call their web :)
_Sample usage at HitBTC site: see https://hitbtc.com/terminal, My Orders tab. Click Cancel button in required order line._

Sure, ran:
buyOrder = exchange.create_limit_buy_order('DOGE/BTC', 1000, 0.000000392)
print(buyOrder['id'])
print(exchange.cancel_order(buyOrder['id']))

orderNotFound and have it alive in the terminal.

**Comment by kroitor** [comment]
Can you please try the following:

```Python
buyOrder = exchange.create_limit_buy_order ('DOGE/BTC', 1000, 0.000000392)
id = buyOrder['info']['ExecutionReport']['clientOrderId'] # we will try this one instead of orderId
print(id)
print(exchange.cancel_order(id))
```

Does it work? If so, then it means they only use client ids, not their own ids to cancel orders.

**Comment by TearsOfTheEarth** [comment]
Yes, works this way 👍

**Comment by kroitor** [comment]
Ok, thx, we will have to fix it for HitBTC then, such a strange API requirements they have... Will update you soon on this.

**Comment by kroitor** [comment]
Ok, this should be fixed and working normally in 1.4.32+.
Please update to the most recent version and use it as you did with your original example:

```Python
buyOrder = exchange.create_limit_buy_order('DOGE/BTC', 1000, 0.000000392)
print(buyOrder['id'])
print(exchange.cancel_order(buyOrder['id'])) # this should now work in 1.4.32+
```

Thank you very much for reporting this!

**Comment by kroitor** [comment]
Sorry, one more minor fix for this issue is coming up in 1.4.33. Please update to 1.4.33 in a couple of minutes.

**Comment by kroitor** [comment]
Done! Fixed in 1.4.33! Thx again!
